### PR TITLE
Create README.md for api docs root folder

### PR DESCRIPTION
**Pull request by wesfloyd**
Creating a simple README to give context to users on the content and purpose of the API docs.

## Why are these changes needed?

The reader needs some context on the purpose and layout of the API docs.

## Checks

Not applicable for this single README file

### CODE REVIEW

File: api/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This folder contains the API documentation for the gRPC services included in the EigenDA platform. Each markdown file contains the protobuf definitions for each respective service including:
+- Churner: a hosted service responsible for maintaining the active set of Operators in the EigenDA network based on their delegated TVL.
+- Disperser: the hosted service and primary point of interaction for Rollup users.
+- Node: individual EigenDA nodes ran on the network by EigenLayer Operators.
+- Retriever: a hosted service that retrieves blobs from EigenDA nodes.
```
Comment (jianoaix) [inline api/docs/README.md:5]: The Retriever isn't hosted, it's up to users to operationalize.

**Review by jianoaix — APPROVED** [review]

**Comment by wesfloyd** [comment]
Thanks @jianoaix - should I contact anyone to ask for this pr to be merged?

**Comment by jianoaix** [comment]
> Thanks @jianoaix - should I contact anyone to ask for this pr to be merged?

can you address the comment?

**Comment by wesfloyd** [comment]
> > Thanks @jianoaix - should I contact anyone to ask for this pr to be merged?
> 
> can you address the comment?

Yes - apologies, will fix that now. Sorry - Github n00b mistake ;)

File: api/docs/README.md (hunk 1 above)
Comment (wesfloyd) [reply api/docs/README.md:5]: Can you share more context on what we mean by "users operationalize"?

E.g. should we say "a service that users can run on their own infrastructure, which exposes a gRPC endpoint for retrieval of blobs from EigenDA nodes"?

**Review by jianoaix — APPROVED** [review]

**Pull request merged**